tests/e2e/README.md

```diff
@@ -0,0 +1,25 @@
+# End-to-End Testing
+
+## Context
+
+The FSx OpenZFS CSI driver utilizes a variety of workflows to validate end to end functionality in a Kubernetes environment.
+The framework and existing testsuites for Kubernetes storage E2E testing is leveraged to test common cases.
+Untested or unique test cases are also tested using a custom testsuite.
```

Comment:
I'd specify the specific workflow the custom test suite is testing

Reply:
I chose not to since we may in the future create other custom test cases for different workflows. If we feel strongly that we should specify the custom cases I can do so in a new section.